.gitpod.yml

```diff
@@ -0,0 +1,28 @@
+tasks:
+  - name: Serve Sphinx Docs
+    init: |
+      cd /workspace/aac-gherkin/docs/

```

Comment:
This should be referencing the repo as part of the path, so should be `aac-gen-gherkin`